var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 # -----------------------------------------------------------------------------
-# Author: Justin Too
+# Author: Nathan Pinnow
 # -----------------------------------------------------------------------------
```

Comment:
This comment isn't actually used by anything, would you like to remove it and add yourself under:
```python
maintainers = ['pinnown']
```
within the class definition? See bullet point 3 under https://spack.readthedocs.io/en/latest/packaging_guide.html#creating-editing-packages for a description of what this means.

Reply:
Switched authors to maintainers.